scripts/tests/test_typos_rollout_check.py

```
@@ -0,0 +1,121 @@
+            checker.load_policy(tmp_path)
+
+    def test_checker_preserves_boundaries_masking_and_exclusions(
+        self, checker: types.ModuleType, tmp_path: Path
+    ) -> None:
+        """Report phrases only when boundaries and policy allow them."""
+        initialize(
+            tmp_path,
+            {
+                "README.md": (
+                    f"{PROHIBITED}\n{TITLE_PROHIBITED} prose\n"
+                    + "pre-hand"
```

Ruff (0.15.21)

[warning] Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)